src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/ExpectedOutputs/SARIF2012.ProvideRuleProperties_Invalid.sarif

```diff
@@ -120,7 +120,7 @@
                   "index": 0
                 },
                 "region": {
-                  "startLine": 22,
```

Comment:
>22 [](http://example.com/codeflow?start=31&length=2)

we faced a similar issue in this PR:
[https://github.com/microsoft/sarif-sdk/pull/2420/files#diff-277d16bd3a7355417dd787cfc0961db73b7f32e73ad49650468d4f4bb6e5c477](https://github.com/microsoft/sarif-sdk/pull/2420/files#diff-277d16bd3a7355417dd787cfc0961db73b7f32e73ad49650468d4f4bb6e5c477)

We should investigate this.

Reply:
Yes, let's open an issue on it.